precompiles/batch/src/tests.rs

```diff
@@ -1034,11 +1038,17 @@ fn batch_not_callable_by_smart_contract() {
 			}
 			.into();
 
-			assert_ok!(RuntimeCall::Evm(evm_call(Alice, input)).dispatch(RuntimeOrigin::root()));
-
-			// batch failed so state is same
-			assert_eq!(balance(Alice), 10_000);
-			assert_eq!(balance(Bob), 0);
+			match RuntimeCall::Evm(evm_call(Alice, input)).dispatch(RuntimeOrigin::root()) {
```

Comment:
We changed the functionality here https://github.com/paritytech/frontier/pull/905 to disallow evm calls from precompiles and smart contracts alike. This assertion is weird in general since the lines below verify that the batch failed, but the evm call succeeds.